docs/rules/attribute-hyphenation.md

```diff
@@ -38,3 +38,17 @@ Default casing is set to `always`
 ```html
 <MyComponent my-prop="prop"/>
 ```
+
+### `[2, "never", { 'ignore': ['data-', 'aria-', 'slot-scope', 'custom-prop'] }]` - Don't use hyphenated name but allow custom attributes
```

Comment:
I wonder whether it wouldn't be easier for users to just merge their settings with default ones. I can't imagine a case where you'd want to remote any of the default settings - they are sort of required for this rule to work properly. So I'd just assume these setting are always there and whatever user adds here - will be merged to the default array.
What do you think about it?

Reply:
I was providing more flexibility for long term use if things change, but I do agree that it's unlikely people will want to overwrite these defaults. I have updated it so it will not overwrite the default array but instead `push` to it.